Credentials/credentials_student_addProcess.php

```diff
@@ -71,21 +55,20 @@
         } else {
             //Write to database
             try {
+                $credentialsCredentialGateway = $container->get(CredentialsCredentialGateway::class);
                 $data = array('gibbonPersonID' => $gibbonPersonID, 'credentialsWebsiteID' => $credentialsWebsiteID, 'username' => $username, 'password' => $passwordFinal, 'notes' => $notes, 'gibbonPersonID' => $gibbonPersonID, 'gibbonPersonIDCreator' => $_SESSION[$guid]['gibbonPersonID'], 'timestampCreator' => date('Y-m-d H:i:s', time()));
-                $sql = 'INSERT INTO credentialsCredential SET credentialsWebsiteID=:credentialsWebsiteID, username=:username, password=:password, notes=:notes, gibbonPersonID=:gibbonPersonID, gibbonPersonIDCreator=:gibbonPersonIDCreator, timestampCreator=:timestampCreator';
-                $result = $connection2->prepare($sql);
-                $result->execute($data);
-            } catch (PDOException $e) {
+                $AI = $credentialsCredentialGateway->insertCredentialsCredential($data);
```

Comment:
Gateway classes with the `use TableAware;` trait have some helper methods for insert, update, and delete.